types/gulp/index.d.ts

```diff
@@ -14,6 +14,7 @@ declare type Strings = string|string[];
 
 declare namespace gulp {
     interface Gulp extends Orchestrator {
+        task(name: string): never;
```

Comment:
Because Gulp extends Orchestrator, and Orchestrator's task() definition is different. 
See https://github.com/gulpjs/gulp/blob/master/index.js#L14
Without this `task(name: string): never;` line, the TypeScript compiler gives an error about the method signatures on line 28 and 39 differing from the Orchestrator.task() signature.